static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -281,6 +290,18 @@ define([
                 callbacks[size] && callbacks[size](event, $slot);
             }
         },
+        allAdsRendered = function (slotId) {
+            forEach(slots, function (value, key) {
+                if (key === slotId) {
+                    slots[slotId].isRendered = true;
+                    return;
+                }
+            });
```

Comment:
you're looping, but not using `value` or `key` anywhere, i.e. all of this is this same as

```
slots[slotId].isRendered = true;
```